katsdpingest/utils.py

```diff
@@ -20,7 +20,7 @@ def cbf_telstate_view(telstate: katsdptelstate.TelescopeState,
 
     Returns
     -------
-    view : :class:`katsdptelstate.TelescopeState`
+    view
```

Comment:
Why remove the type info here? Is it superfluous given the type annotations?

Reply:
Exactly. Saves me having to maintain it in two places, and modern versions of Sphinx will show the signature.

Reply:
Yay!